waffle-compiler/test/flattener/flattener.ts

```diff
@@ -0,0 +1,41 @@
+import {expect} from 'chai';
+import fs from 'fs-extra';
+import {inputToConfig} from '../../src/config';
+import {flattenAndSave} from '../../src';
+
+const sourceDirectory = './test/flattener/testSource';
+const flattenOutputDirectory = './test/flattener/flattenFiles';
+const compilerVersion = 'v0.5.9+commit.e560f70d';
+const config = inputToConfig({
+  sourceDirectory,
+  flattenOutputDirectory,
+  compilerVersion
+});
+
+const flatAndGetChild = async () => {
+  await flattenAndSave(config);
+  return fs.readFileSync('./test/flattener/flattenFiles/child.sol', 'utf8');
+};
+
+describe('flattening', () => {
+  after(async () => {
+    await fs.remove(flattenOutputDirectory);
```

Comment:
This is very problematic together with #tmpoutput, as after test run, you will have changes to stage in git.
Remove `waffle-compiler/test/flattener/flattenFiles` from repo and add it to `.gitignore` to fix it.